docs/book/src/cronjob-tutorial/testdata/project/.github/workflows/lint.yml

```diff
@@ -20,4 +20,4 @@ jobs:
       - name: Run linter
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.61
+          version: v1.61.0
```

Comment:
To simplify updates, we could get version from Makefile instead. What you think?